src/OpenTelemetry.Exporter.Jaeger/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   were sent as `CHILD_OF`.
   ([#970](https://github.com/open-telemetry/opentelemetry-dotnet/pull/970))
 
+* Rename extension method `UseJaegerExporter` to `AddJaegerExporter`.
```

Comment:
I think we need to adding the missing `0.4.0-beta.2` to the log.
I would vote for using passive voice for consistency.